test/webhooks/k8ssandracluster_webhook_test.go

```diff
@@ -235,6 +243,120 @@ func testNumTokens(t *testing.T) {
 	require.Error(err)
 }
 
+func testTelemetryValidation(t *testing.T) {
```

Comment:
This should be ported to envtest as this test was envtest and there's nothing in this test that would not work in a envtest. At the moment this test does not verify anything, since it's not targeting Kubernetes API.

Reply:
As discussed, I do not believe an envtest is required for this change and do not intend to write one.

Reply:
Then this is considered as non-tested code and should not be merged.

Reply:
Considering there is a test which explicitly says `testTelemetryValidation` your comment above is clearly untrue @burmanm, you should consider withdrawing it as it makes you look silly.